client/client/app/components/ngbBlastSearchPanel/ngbBlastWholeGenomeView/ngbBlastWholeGenomeView.dialog.tpl.html

```diff
@@ -0,0 +1,26 @@
+<md-dialog
+    aria-label="whole genome view"
+    style="width: 66%"
+>
+    <md-toolbar>
+        <div class="md-toolbar-tools">
+            <h3>BLAST results: whole genome view</h3>
+            <span flex></span>
+            <md-button
+                class="md-icon-button"
+                ng-click="close()"
+            >
+                <ng-md-icon
+                    style="fill: #ffffff;"
+                    icon="clear"
+                    aria-label="Close dialog"
+                ></ng-md-icon>
+            </md-button>
+        </div>
+    </md-toolbar>
+    <ngb-blast-whole-genome-view
+        chromosomes = "chromosomes"
+        max-chr-size = "maxChrSize"
```

Comment:
Don't forget to remove `max-chr-size` after refactoring the calculation (moving to the `<ngb-blast-whole-genome-view>`)